docker-compose.yml

```
@@ -370,6 +370,22 @@ services:
       - "9927:80"
       - "19927:19927"
 
+  thoracic:
+    <<: *node
+    <<: *site-cmd
+    working_dir: /root/sites/thoracicam.com
+    environment:
+      <<: *env
+      <<: *env-tauron
+      PORT: 80
+      EXPOSED_PORT: 9928
+      LIVERELOAD_PORT: 19928
+      TENANT_KEY: ascend_hh
+      SITE_ID: 62f156eeb42f9308288b45e8
+    ports:
```

All BaseCMS website service definitions should now use the `hostname` service key, and include the `HOST` environment variable, to ensure a consistently unique cookie domain across environments. This ensure both that the IdentityX, Omeda, and P1Events contexts (where applicable) will be correct when switching between sites.

```suggestion
      SITE_ID: 62f156eeb42f9308288b45e8
      HOST: www-ascend-thoracic.dev.parameter1.com
    hostname: www-ascend-thoracic.dev.parameter1.com
    ports:
```